### DISCUSSION TITLE
dialect:sqlite Thoughts and Questions

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1153] [General]
Perhaps we should use this topic to catalog things specific to SQLite. (And other discussion topics could serve for other dialects...)

- You probably know this, but SQLite doesn't support CONCAT(). It uses `||` between items. See [https://www.sqlitetutorial.net/sqlite-string-functions/sqlite-concat/](https://www.sqlitetutorial.net/sqlite-string-functions/sqlite-concat/)

**Comment by max-sixty** [comment]
Great, +1 for collecting and prioritizing dialect issues. The important outcome will be the top things to implement.

As dialect-specific implementations grow, we'll likely need to consider https://github.com/prql/prql/issues/873 — does translation become a trait which various dialects implement?
